src/environments/environment.ts

```diff
@@ -67,3 +67,9 @@ export const decorateModuleRef = _decorateModuleRef;
 export const ENV_PROVIDERS = [
   ...PROVIDERS
 ];
+
+export const environment = {
+  production: false,
```

Comment:
Why is this needed?

Reply:
This was somehow included in the Rebase. I'll remove it.